Updating style on Smart Cities? Public Code! to match the rest of our content (#30)
publiccodenet/smart-cities-public-code: index.md

We want to do this by compiling a first edition of a Guide for the Deployment of Public Software that documents steps and issues involved in the deployment of public software.
This guide will be produced in co-creation sessions in three international workshop. The discussions about the creation of this guide will also be used to draft a research agenda. What steps need to be taken next in terms of research and development for the further propagation of public software?

Through these workshops and its subsequent publication of the Guide and the Research Agenda we aim to connect research institutions, municipalities and higher level government policy makers and creative industry companies around this important theme and stimulate further cooperation in this field.

As such, the scope of this project is mainly an agenda-building one, that aims to identify main stakeholders, list the most important issues, build a research agenda and set up a practical guide based on our knowledge so far.

## Relevance in relation to the [Dutch national research agenda](https://wetenschapsagenda.nl/national-science-agenda/?lang=en)

> This topic is relevant for various aspects of both the research and innovation agenda, as well as the smart culture roadmap.

Author: How much scope for reworking this is there? If this sentence is reworked as I suggest below, then the empty H3 could be deleted, and the H4s all bumped up a level to H3.

```suggestion
This topic is relevant for the following goals on smart culture roadmap and the research and innovation agenda.
```
